lib/commands/source.js

```diff
@@ -10,8 +10,8 @@ const APPIUM_SRC_XML = '<?xml version="1.0" encoding="UTF-8"?><AppiumAUT/>';
 // is fixed, we need to translate the request
 commands.getPageSource = async function () {
   if (this.isWebContext()) {
-    let cmd = 'document.getElementsByTagName("html")[0].outerHTML';
-    return await this.remote.execute(cmd);
+    const script = 'return document.documentElement.outerHTML';
```

Comment:
is _return_ needed there?

Reply:
I don't understand what you mean? There is a return on line 14.

Reply:
I mean the previous cmd had not return : `'document.getElementsByTagName("html")[0].outerHTML'`

Reply:
Oh! Haha. Yes, that is necessary because we are running the execute_script atom. The old execute worked differently.